src/Calc.php

```diff
@@ -22,8 +22,24 @@ public function sayHello(DateTimeImmutable $date): void
         echo 'Hello, ' . $date->format('j. n. Y');
     }
 
+    public $displayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaaayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaa;
+
+    public function sayaHello(DateTimeImutable $date): void
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Parameter $date of method Acme\Calc::sayaHello() has invalid typehint type Acme\DateTimeImutable.